gtsam/geometry/Rot3.cpp

```diff
@@ -197,27 +197,15 @@ Matrix3 Rot3::LogmapDerivative(const Vector3& x)    {
 /* ************************************************************************* */
 pair<Matrix3, Vector3> RQ(const Matrix3& A) {
 
-  // atan2 has curious/unstable behavior near 0.
-  // If either x or y is close to zero, the results can vary depending on
-  // what the sign of either x or y is.
-  // E.g. for x = 1e-15, y = -0.08, atan2(x, y) != atan2(-x, y),
-  // even though arithmetically, they are both atan2(0, y).
-  // Suppressing small numbers to 0.0 doesn't work since the floating point
-  // representation still causes the issue due to a delta difference.
-  // The only fix is to convert to an int so we are guaranteed the value is 0.
-  // This lambda function checks if a number is close to 0.
-  // If yes, then return the integer representation, else do nothing.
-  auto suppress = [](auto x) { return abs(x) > 1e-15 ? x : int(x); };
-
-  double x = -atan2(-suppress(A(2, 1)), suppress(A(2, 2)));
+  double x = -atan2(-A(2, 1), A(2, 2));
```

Comment:
Uhm, please add more justification than “I removed it’ ;-)

Reply:
This is an undo of a previous commit I had introduced in this PR. The stability is less of a problem compared to the behavior of `Eigen::Transpose`.